macros/src/main/scala/gestalt/macros/Optional.scala

```diff
@@ -28,11 +28,14 @@ final class Optional[+A >: Null](val value: A) extends AnyVal {
         Select(tempIdent, "value")
     }
 
-    q"""
+    {
+      import scala.gestalt.options.unsafe
+      q"""
        $tempValDef
        if ($tempIdent.isEmpty) new Optional(null)
-       else new Optional($newBody)
+       else new Optional(${newBody.wrap})
```

Comment:
This is common with API arguments that are `List`s. It will happily create the wrong type of List - `List[tpd.Tree]` here, but it can also can create `List[AnyRef]` when there is a mixture. Then it cannot find a method with such signature and fails.